Mathlib/CategoryTheory/Square.lean

```diff
@@ -146,8 +146,7 @@ def isoMk {sq₁ sq₂ : Square C} (e₁ : sq₁.X₁ ≅ sq₂.X₁) (e₂ : sq
 
 /-- Flipping a square by switching the top-right and the bottom-left objects. -/
 @[simps]
-def flip (sq : Square C) : Square C where
-  fac := sq.fac.symm
+def flip (sq : Square C) : Square C := { fac := sq.fac.symm, .. }
```

Comment:
This one makes me sad...

Reply:
I added the inferable fields. Not sure if it's better that way...